lib/stagehand/staging/commit.rb

```diff
@@ -5,13 +5,16 @@ def self.all
         CommitEntry.start_operations.pluck(:id).collect {|id| find(id) }
       end
 
-      def self.capture(subject_record = nil, &block)
+      def self.capture(*args, &block)
```

Comment:
What if this was keyword args? `def self.capture(subject_record = nil, except: [], &block)`. Then it makes it obvious what the options can be, since `end_commit` is private and deeper down in the file.